auto_submit/lib/validations/revert.dart

```diff
@@ -30,21 +30,39 @@ class Revert extends Validation {
     final auto.Commit commit = pullRequest.commits!.nodes!.single.commit!;
     final String? sha = commit.oid;
 
+    final github.RepositorySlug slug = messagePullRequest.base!.repo!.slug();
+    GithubService githubService = await config.createGithubService(slug);
+    final github.PullRequest updatedPullRequest = await githubService.getPullRequest(
+      slug,
+      messagePullRequest.number!,
+    );
+
     if (!isValidAuthor(author, authorAssociation)) {
       final String message = 'The author $author does not have permissions to make this request.';
       log.info(message);
       return ValidationResult(false, Action.REMOVE_LABEL, message);
     }
 
-    final bool? canMerge = messagePullRequest.mergeable;
-    if (canMerge == null || !canMerge) {
+    final bool? canMerge = updatedPullRequest.mergeable;
+
+    if (canMerge == null) {
```

Comment:
Will this only be null or true? Can this be false?

Reply:
The value can be null, true or false. The false case is handled in the code below. If the value is null then github is still calculating the mergeability, if true it can be merged if false then there are conflicts.